### ISSUE TITLE
Timetable colour contrast in Night mode

### ISSUE THREAD

**Issue by KavishaniVimalathasan** [tri-tim-m-nguyen/study-plan-application#28]
The colour contrast of the create timetable page has to be improved in night/dark mode, as the list of time slots looks entirely blacked out in night mode.

**Comment by RayanTarawneh** [comment]
This PR #40 addresses and fixes this bug
